project/Doc.scala

```diff
@@ -166,7 +166,7 @@ object BootstrapGenjavadoc extends AutoPlugin {
   override def requires =
     UnidocRoot.CliOptions.genjavadocEnabled
       .ifTrue {
-        val onJdk8 = System.getProperty("java.version").startsWith("1.")
+        val onJdk8 = JavaVersion.is1x
         require(!onJdk8, "Javadoc generation requires at least jdk 11")
```

Comment:
@johanandren @raboof Shouldn't it actually be more like: if java.specification.version < 11 based on the requirement message or is it actually just requiring 8 and the message needs to be updated?

Reply:
I think we only care about 8 and 11 at the moment

Reply:
`version > 9` I think but since that means not 8, I think that is fine.